tests/utils/verification/volumeproperties.go

```diff
@@ -55,42 +55,27 @@ func GetVMAttachedToVolUsingAdminCli(volName string, hostname string) string {
 	return volProps[1]
 }
 
-// GetVolumePropertiesAdminCli returns capacity, attached-to-vm and disk-format field
-// for volume using Admin cli
-func GetVolumePropertiesAdminCli(volName string, hostname string) string {
-	cmd := admincli.ListVolumes + "-c volume,attached-to,capacity,disk-format 2>/dev/null | grep " + volName
-	op, _ := ssh.InvokeCommand(hostname, cmd)
-	if op == "" {
-		log.Fatal("Null value is returned by admin cli when looking for, size, disk-format and attached to vm. Output: ", op)
-	}
-	if len(strings.Fields(op)) != 4 {
-		log.Fatalf("Output is expected to consist of four elements only - "+
-			"volume name, attached-to-vm status, size and disk-format. Actual output: %s", op)
+// GetVolumePropertiesAdminCli returns a map of volumes and their corresponding
+// properties - capacity, attached-to-vm and disk-format field using Admin cli
+func GetVolumePropertiesAdminCli(hostName string) map[string][3]string {
+	log.Printf("Getting size, disk-format and attached-to-vm for all volumes from ESX [%s] using admin cli. \n", hostName)
+	cmd := admincli.ListVolumes + "-c volume,capacity,disk-format,attached-to 2>/dev/null | awk -v OFS='\t' '{print $1, $2, $3, $4}' | sed '1,2d'  "
+	out, _ := ssh.InvokeCommand(hostName, cmd)
+	admincliValues := strings.Fields(out)
+	adminCliMap := make(map[string][3]string)
+	for i := 0; i < len(admincliValues); {
+		adminCliMap[admincliValues[i]] = [3]string{admincliValues[i+1], admincliValues[i+2], admincliValues[i+3]}
+		i = i + 4
 	}
-	return op
+	return adminCliMap
```

Comment:
Its global in a test file tests/e2e/volumeproperty_test.go and not under tests/utils/verification/volumeproperties.go